### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic

### ISSUE THREAD

**Issue by Riobe** [mogelbrod/apply-loader#5]
### Problem
When using webpack 2 with apply-loader and an options object, like so:

```javascript
/ Handle pug transpiling
rules.push({
  test: /\.pug$/,
  loaders: [
    {   
      loader: 'apply-loader',
      options: {
        obj: {
          ENV: options.prod ? 'production' : 'development'
        }   
      }   
    },  
    {   
      loader: 'pug-loader',
      options: {
        globals: {
        }   
      }   
    }   
  ]
});
```
The following error is thrown twice:

```bash
(node:29204) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

### Versions

* Node: 6.10.2
* npm: 3.10.10
* webpack: 2.4.1
* apply-loader 1.0.0

```bash
$ uname -srvpio
Linux 3.10.0-514.16.1.el7.x86_64 #1 SMP Wed Apr 12 15:04:24 UTC 2017 x86_64 x86_64 GNU/Linux
```

**Comment by mogelbrod** [comment]
Try the the newly released v2.0.0 👍

**Comment by Riobe** [comment]
I think that fixed it! I'm still getting the warning in my build, but I went into your index.js and added a `console.log` at the beginning and right before the return of `module.exports.pitch`, and there's no warnings being thrown in-between those lines so I think it must be from another loader. From what I can see apply-loader is good to go (and super useful in my case).

Thank you very much for the fix, and for getting to it so fast!
